packages/system/src/components/landing/resources-info-banner.css.ts

```diff
@@ -1,20 +1,19 @@
 import { style } from "@vanilla-extract/css"
 import { breakpoints, sprinkles } from "../../sprinkles/sprinkles.css"
-import { vars } from "../../themes/themes.css"
 import { pxToRem } from "../../util/style-utils"
 
 export const resourcesInfoBannerStyle = style([
 	sprinkles({
 		borderRadius: 12,
 		layout: "stack",
 		gap: 20,
-		padding: { desktop: 56, mobile: "auto" },
+		paddingY: { desktop: 56, mobile: "auto" },
+		paddingX: { desktop: 56, mobile: "auto" },
 	}),
 	{
-		maxWidth: pxToRem(1080),
 		"@media": {
 			[breakpoints.desktop]: {
-				backgroundColor: vars.palette.neutralVariant99,
+				backgroundColor: "var(--banner-background)",
```

Comment:
Why aren't we using a var here instead?

Reply:
Oh that's because now that value comes from a prop in the component!